.github/workflows/integrate.yaml

```diff
@@ -6,7 +6,7 @@ name: CI
 
 on:
     push:
-      branches: [ master ]
+      branches: [ * ]
```

Comment:
@eltonn o CI parece ter parado de funcionar, qual a necessidade de modificar este parâmetro aqui?
Foram feitos pushes para branches com PRs em master desde dentro do upstream ao qual o actions funcionou corretamente.

Reply:
creio que a ideia disso é ter o CI funcionando com os secrets dentro dos branches.

Reply:
Sorry, não entendi... o secrets esta configurada dentro deste repositório, qualquer branch (no repo) que contenha a chamada no yaml das variáveis de ambiente funcionaria, do contrário, os outros PRs anteriores após a implementação do secrets aqui não teriam tido susseso.

Reply:
mas pra isso vc precisaria abrir o PR pra ativar o CI, correto? se vc tem q ter o branch no repositorio obrigatoriamente pra poder rodar o secrets, vc tb poderia ter o CI rodando ... assim vc soh abre o PR qndo tiver algo pronto pra mandar

Reply:
Correto o Ci é ativado quando é feito push or PR para master, a finalidade então é que se execute o workflow nada mais subir o branch para o repo upstream ou a cada push para o mesmo, talvez usar algum outro evento que dispara o webhook ativando o workflow:
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows

Temos de ficar atentos para não executar mais de uma vez o CI no push quando esteja aberto o PR.

Reply:
se o CI nao funciona pra PRs de forks .. talvez seja melhor entao remover o trigger pra PR, pelo menos pro downloadapp, e deixar soh pra push para todos os branches